app/javascript/client/cart/CartProductThumbnail.tsx

```diff
@@ -163,27 +187,28 @@ export const CartProductThumbnail: FC<CartProductThumbnailProps> = ({ product, q
             {({ isSubmitting }) => (
                 <Form>
                     <ItemContainer>
-                        <ImageContainer>
-                            {imageUrl ? (<Image src={imageUrl} alt={name} />) : (<ImageStandIn>The Tree Company sign</ImageStandIn>)}
-                        </ImageContainer>
-                        <CartProductDetails>
-                            <ProductNameContainer>{name}</ProductNameContainer>
-                            <div>
-                                Size: <br />{size}
-                                <br />
-                                <br />
-                                Material: <br />{material}
-                            </div>
-                            {counties && <div>Counties: <br />{countyList}</div>}
-                        </CartProductDetails>
+                        <ImageAndDetailContainer>
+                            <ImageContainer>
+                                {imageUrl ? (<Image src={imageUrl} alt={name} />) : (<ImageStandIn>The Tree Company sign</ImageStandIn>)}
+                            </ImageContainer>
+                            <CartProductDetails>
+                                <ProductNameContainer>{name}</ProductNameContainer>
+                                <ProductCartInfo>
+                                    <span>Size: <br />{size}</span>
+                                    <span>Material: <br />{material}</span>
```

Comment:
I think it's a bit of a best practice use CSS (i.e., margin or padding) for spacing instead of `<br/>`s ?? NBD to me, just an FYI
https://www.reddit.com/r/web_design/comments/34y8b9/using_br_to_separate_elements_instead_of/